src/lib.rs

```diff
@@ -77,6 +77,7 @@ pub struct AdapterOptions {
     pub async_init: bool,
     pub compression: bool,
     pub invoke_mode: LambdaInvokeMode,
+    pub authorization_source: String,
```

Comment:
It would be better to use `Option<String>` as type. Similiar to `base_path`.

Reply:
Thanks, fixed!